code/modules/clothing/suits/labcoat.dm

```diff
@@ -63,16 +63,34 @@
 	desc = "A suit that protects against minor chemical spills. Has an orange stripe on the shoulder."
 	icon_state = "labcoat_chem"
 
+/obj/item/clothing/suit/toggle/labcoat/chemist/Initialize(mapload)
+	. = ..()
+	allowed += list(
+		/obj/item/storage/bag/chemistry,
+	)
```

Comment:
why not just allowed += /obj/item/storage/bag/chemistry, same with the other things in this file

Reply:
Tried to do it that way but keep getting error with: "got '+-', expected one of: '/', contents, newline, identifier". So decided to do it like it was done to cmo winter coat

Reply:
You probably got the syntax wrong. Here's a suggestion to help you:
```suggestion
	allowed += /obj/item/storage/bag/chemistry
```